control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -135,7 +135,7 @@ func ReconcileDeployment(deployment *appsv1.Deployment, ownerRef config.OwnerRef
 					util.BuildContainer(cvoContainerBootstrap(), buildCVOContainerBootstrap(cliImage, clusterID)),
 				},
 				Containers: []corev1.Container{
-					util.BuildContainer(cvoContainerMain(), buildCVOContainerMain(image)),
+					util.BuildContainer(cvoContainerMain(), buildCVOContainerMain(controlPlaneImage)),
```

Comment:
shouldn't buildCVOContainerMain receive both the controlPlaneImage and the userImage so then it runs the pod from the former but it points --release-image to the latter?

Reply:
nice find, thanks :+1: